From b278ad5f3c10be9b5feb719e2db4f12f24e3b75b Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 5 Sep 2020 10:46:05 +0200
Subject: [PATCH 1/1] doc: describe source repository

Add a chapter to the HTML documentation describing how to retrieve the
U-Boot sources.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/build/index.rst  |  1 +
 doc/build/source.rst | 30 ++++++++++++++++++++++++++++++
 2 files changed, 31 insertions(+)
 create mode 100644 doc/build/source.rst

diff --git a/doc/build/index.rst b/doc/build/index.rst
index e0072afb5e..5051a97e70 100644
--- a/doc/build/index.rst
+++ b/doc/build/index.rst
@@ -6,5 +6,6 @@ Build U-Boot
 .. toctree::
    :maxdepth: 2
 
+   source
    clang
    tools
diff --git a/doc/build/source.rst b/doc/build/source.rst
new file mode 100644
index 0000000000..4627c1f79e
--- /dev/null
+++ b/doc/build/source.rst
@@ -0,0 +1,30 @@
+Obtaining the source
+=====================
+
+The source of the U-Boot project is maintained in a Git repository.
+
+You can download the source via
+
+.. code-block:: bash
+
+    git clone https://gitlab.denx.de/u-boot/u-boot.git
+
+A mirror of the source is maintained on Github
+
+.. code-block:: bash
+
+    git clone https://github.com/u-boot/u-boot
+
+The released versions are available as tags which use the naming scheme::
+
+    v<year>.<month>
+
+Release candidates are named::
+
+    v<year>.<planned relase month>-rc<number>
+
+To checkout the October 2020 release your would use
+
+.. code-block:: bash
+
+    git checkout v2020.10
-- 
2.28.0

